include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3303,6 +3303,31 @@ struct OrtApi {
   */
   ORT_API2_STATUS(SessionOptionsAppendExecutionProvider_MIGraphX,
                   _In_ OrtSessionOptions* options, _In_ const OrtMIGraphXProviderOptions* migraphx_options);
+
+  /** \brief Replace initialized Tensors with external data with the data provided in inputs.
```

Comment:
>inputs [](http://example.com/codeflow?start=86&length=6)

nit: could the name "inputs" be confusing? maybe "initializer_names"/"initializer_values" or "names"/"ort_values" (like the C++ API)